pom.xml

```diff
@@ -21,12 +21,12 @@
     <system>GitHub</system>
     <url>https://github.com/NationalSecurityAgency/emissary/issues</url>
   </issueManagement>
-  <ciManagement />
-  <distributionManagement />
+  <ciManagement></ciManagement>
+  <distributionManagement></distributionManagement>
```

Comment:
It's weird that these are expanded, since the plugin is configured to not expand them.

Reply:
suggests that the configuration isn't taking effect perhaps? I'll need to look closely.